_posts/2016-01-01-Markdown Reference.md

```diff
@@ -345,46 +352,46 @@ Use the `printf()` function.
 
 ### Strikethrough
 
-GFM adds syntax to create strikethrough text, which is missing from standard Markdown.
+GFM adds syntax to create strikethrough text, which is missing from standard Markdown. <!-- Comment: The same goes for tables, footnotes etc. but you've not mentioned that then. -->
 
 `~~Mistaken text.~~` becomes ~~Mistaken text.~~
 
-### Underlines
-
-Underline is powered by raw HTML.
-
-`<u>Underline</u>` becomes <u>Underline</u>.
-
 ### Emoji :happy:
 
-Input emoji with syntax `:smile:`.
+Enter emoji with syntax `:smile:`.
 
-User can trigger auto-complete suggestions for emoji by pressing `ESC` key, or trigger it automatically after enabling it on preference panel. Also, inputting UTF-8 emoji characters directly is also supported by going to `Edit` -> `Emoji & Symbols` in the menu bar.
+User can trigger auto-complete suggestions for emoji by pressing `ESC` key<!-- it doesn't for me -->, or trigger it automatically after enabling it on preference panel <!--where's that? -->. Also, entering UTF-8 emoji characters directly is also supported by going to `Edit` -> `Emoji & Symbols` in the menu bar.
```

Comment:
> User can trigger auto-complete suggestions for emoji by pressing `ESC` key<!-- it doesn't for me -->, or trigger it automatically after enabling it on preference panel <!--where's that? -->.

This is old spec, now emoji autocomplete will be displayed automatically after input `:` and keyword.

Reply:
OK, have updated.